python_modules/dagster/dagster_tests/core_tests/definitions_tests/test_repository_definition.py

```diff
@@ -85,5 +85,11 @@ def noop2():
         },
     )
 
-    with pytest.raises(DagsterInvalidDefinitionError):
+    with pytest.raises(DagsterInvalidDefinitionError) as exc_info:
         repo.get_all_pipelines()
+
+    assert str(exc_info.value) == (
```

Comment:
im not a huge fan of exact text match in these tests, not sure what a good solution is that solves the same problem

Reply:
it's a pain but I've found the practice improves error message quality